examples/pygi_mpl_numpy/README.rst

```diff
@@ -73,6 +73,8 @@ When the pygi-aio bundle is installed on a Windows-machine, the installer perfor
 
 An example are the glib-schemas which are required for the GtkFileChooserDialog to work properly. The script ``2_extract.sh`` will automatically call the following command to compile the glib-schemas:
 
+Note: If your application uses glib-schemas (e.g. default-settings stored in gsettings) you will need to place the schemas for your application into that folder and recompile it before packaging it with Pynsist.
+
```

Comment:
Can this go below the command instead of above it? The wording of the paragraph above means that the command should be immediately after it.